components/actionbar/index.css

```diff
@@ -147,27 +123,38 @@
 
 	/* action group */
 	.spectrum-ActionGroup {
-		margin-inline-end: var(--mod-actionbar-spacing-action-group-end, var(--spectrum-actionbar-spacing-action-group-end));
-		margin-block-start: var(--mod-actionbar-spacing-action-group-top, var(--spectrum-actionbar-spacing-action-group-top));
-
 		/* align to end by default */
 		margin-inline-start: auto;
 	}
 }
 
 .spectrum-ActionBar--emphasized {
 	.spectrum-ActionBar-popover {
-		filter: none;
 		background-color: var(--mod-actionbar-emphasized-background-color, var(--spectrum-actionbar-emphasized-background-color));
 
-		/* border transparent instead of none so WHCM will have visible border */
-		border-color: transparent;
+		.spectrum-CloseButton:hover .spectrum-Icon {
+			color: var(--highcontrast-actionbar-closebutton-highlight-color, var(--spectrum-actionbar-emphasized-icon-color));
+		}
 	}
 
 	/* ensure text is legible on emphasized background */
 	.spectrum-FieldLabel {
 		color: var(--mod-actionbar-emphasized-item-counter-color, var(--spectrum-actionbar-emphasized-item-counter-color));
 	}
+
+	.spectrum-ActionGroup .spectrum-ActionButton {
+		.spectrum-ActionButton-label {
+			color: var(--highcontrast-actionbutton-emphasized-content-color-default, var(--spectrum-actionbar-emphasized-action-button-label-color));
+		}
+
+		.spectrum-ActionButton-icon {
+			color: var(--highcontrast-actionbutton-emphasized-icon-color, var(--spectrum-actionbar-emphasized-icon-color));
+		}
+	}
+
+	.spectrum-CloseButton .spectrum-Icon {
+		color: var(--spectrum-actionbar-emphasized-icon-color);
+	}
```

Comment:
I don't think we need to be changing colors on the action buttons or the icons. Or the close button? I didn't see any separate colors defined on the spec (but the specs look unfinished).

I saw this on the S2 desktop Figma "In the action bar, the action buttons use the static white and static black versions, which remain the same between color themes." That might be related. I don't think that's being done currently; we only use one of the static color classes.